src/saml.ts

```diff
@@ -131,7 +131,9 @@ class SAML {
       throw new TypeError("SamlOptions required on construction");
     }
 
-    const options = {
+    assertRequired2(ctorOptions.cert, "cert is required");
```

Comment:
unrelated change at production code (i.e. not some randomish test code change)...see also similar change at line 158/161